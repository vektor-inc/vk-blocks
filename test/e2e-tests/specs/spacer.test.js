/**
 * WordPress dependencies
 */
 import {
    createNewPost,
    getEditedPostContent
} from '@wordpress/e2e-test-utils';

/**
 * Internal dependencies
 */
import { changeSiteLang } from '../helper/changeSiteLang';

describe( 'Spacer', () => {
    let oldLanguage;
	beforeEach( async () => {
        oldLanguage = await changeSiteLang( 'ja' );
		await createNewPost();
	} );
    afterEach( async () => {
        await changeSiteLang( oldLanguage );
    });
    it( 'should insert block name in Japanese', async () => {
        //Using the slash command
        await page.keyboard.press( 'Enter' );
        await page.keyboard.type( '/レスポンシブスペーサー' );
        await page.waitForXPath(
            `//*[contains(@class, "components-autocomplete__result") and contains(@class, "is-selected") and contains(text(), 'レスポンシブスペーサー')]`
        );
        await page.keyboard.press( 'Enter' );
        expect( await getEditedPostContent() ).toMatchSnapshot();
    } )
});