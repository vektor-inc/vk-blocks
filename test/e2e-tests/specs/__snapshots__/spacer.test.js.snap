// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spacer should insert block name in Japanese 1`] = `
"<!-- wp:vk-blocks/spacer -->
<div class=\\"wp-block-vk-blocks-spacer vk_spacer vk_spacer-type-margin-top\\"><div class=\\"vk_block-margin-md\\"></div></div>
<!-- /wp:vk-blocks/spacer -->"
`;
